test/controllers/profiles_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ProfilesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
We tend to not write controller tests in favor of integration tests. I'd say you can safely remove this for now and we can cover integration tests when you're ready.